init_test.go

```diff
@@ -0,0 +1,24 @@
+package radiance
+
+import (
+	"testing"
+
+	"github.com/getlantern/radiance/client"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestInitialize(t *testing.T) {
+	opts := client.Options{
+		DataDir:  "/tmp/data",
+		LogDir:   "/tmp/logs",
+		Locale:   "en-US",
+		DeviceID: "test-device-id",
+	}
```

Comment:
This will ensure that the files/folders are cleaned up when the test finishes.
```suggestion
func TestInitialize(t *testing.T) {
	tmp := t.TempDir()
	opts := client.Options{
		DataDir:  tmp,
		LogDir:   tmp,
		Locale:   "en-US",
		DeviceID: "test-device-id",
	}
```